tests/gdk/common/test_CaseInsensitive.py

```diff
@@ -37,6 +42,24 @@ def test_read_yaml(self):
         assert "uri" in case_insensitive_recipe["Manifests"][0]["Artifacts"][0]
         assert "URI" in case_insensitive_recipe["Manifests"][0]["Artifacts"][0]
 
+    def test_read_json_with_syntax_err(self):
+        self.caplog.set_level(logging.ERROR)
+        json_file = Path(".").joinpath("tests/gdk/static/project_utils").joinpath("recipe_missing_comma.json").resolve()
+        with pytest.raises(Exception) as e:
+            CaseInsensitiveRecipeFile().read(json_file)
+        logs = self.caplog.text
+        assert "For information and examples regarding component recipes refer to the docs here" in logs
+        assert "Expecting ',' delimiter: line 6 column 3" in str(e)
+
+    def test_read_yaml_with_syntax_err(self):
+        self.caplog.set_level(logging.ERROR)
+        json_file = Path(".").joinpath("tests/gdk/static/project_utils").joinpath("invalid_component_recipe.yaml").resolve()
```

Comment:
Since we didn't create this file for this test, are we sure it will remain consistent for this test?

Reply:
nit: it's also not a `json_file`

Reply:
Good catch, I'll update the variable name.